### ISSUE TITLE
Does not detect Time Zone language

### ISSUE THREAD

**Issue by firefoxlover** [Vanilla-OS/vanilla-installer#182]
My language was english but the installer said I should use African something. I guess the first element in an alphanumerical list.

The timezone itself was correctly identified though.

VanillaOS-22.10-all.20230226

**Comment by kbdharun** [comment]
> My language was english but the installer said I should use African something. I guess the first element in an alphanumerical list.

Yes, that is one of the current limitations of our backend distinst, we use automatic detection of timezones but not the language based on keyboard layout. It should probably be fixed with Albius, I will keep this issue open until it is fixed.

**Comment by firefoxlover** [comment]
@deepu9 no its not. Its a problem that the language is not chosen by default, but the first element in alphabetical order.

**Comment by deepu9** [comment]
> > My language was english but the installer said I should use African something. I guess the first element in an alphanumerical list.
> 
> Yes, that is one of the current limitations of our backend distinst, we use automatic detection of timezones but not **the language based on keyboard layout**. It should probably be fixed with Albius, I will keep this issue open until the confirmation.

@firefoxlover I'm referring to the above message. My apologies for the misunderstanding.
